sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -72,6 +72,48 @@ class HiveThriftBinaryServerSuite extends HiveThriftJdbcTest {
     try f(client) finally transport.close()
   }
 
+  test("SPARK-29492: use add jar in sync mode") {
```

Comment:
let's put the new test at the end.

Reply:
can you post the error message when running the test before this patch?

Reply:
> can you post the error message when running the test before this patch?

Done